subgraph_extractor/cli.py

```diff
@@ -5,48 +5,52 @@
 import pandas
 import pyarrow
 import pyarrow.parquet as pq
+import pyarrow.dataset as ds
+from pyarrow import fs
 import yaml
 from cloudpathlib import AnyPath, CloudPath
 from deepdiff import DeepDiff
 from simple_term_menu import TerminalMenu
 from tqdm import tqdm
 import tempfile
 
-TYPE_MAPPINGS = {"numeric": "bytes", "text": "string", "boolean": "bool"}
+
+TYPE_MAPPINGS = {"numeric": "bytes", "text": "string", "boolean": "bool", "bytea": "bytes"}
 
 BLOCK_COLUMN = "_block_number"
-BLOCK_COLUMN_TYPE = pyarrow.uint32()
+BLOCK_COLUMN_TYPE = "uint32"
 
 
-def get_select_all_exclusive(
+def select_writable_in_range(
```

Comment:
This is a fix alongside other changes, I'd made a too-complex query to remove some columns when it's far easier just to pull everything and manually remove the columns in the dataframe.